File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.kt
// Brute Force
// O(n ^ 2)
class Solution {
    fun longestConsecutive(nums: IntArray): Int {
        nums.sort()
        var maxLen = 0
        for(subject in nums) {
            var len = 0
            for(num in nums) {
                if(num == subject + len) {
                    len++
                }
            }
            maxLen = maxOf(maxLen, len)
        }
        return maxLen
    }
}

File: Data Structures & Algorithms/palindromic-substrings/submission-1.cpp
// Brute Force
class Solution {
public:
    int countSubstrings(string s) {
        int count = 0;
        int n = s.size();

        for(int i = 0; i < n; i++) {
            for(int j = i; j < n; j++) {
                int l = i, r = j;
                while(l < r && s[l] == s[r]) {
                    l++;
                    r--;
                }
                if(l >= r) {
                    count++;
                }
            }
        }

        return count;
    }
};